build.sbt

```diff
@@ -878,6 +894,75 @@ lazy val junit = project.in(file("test") / "junit")
     unmanagedSourceDirectories in Test := List(baseDirectory.value)
   )
 
+lazy val dotc = inputKey[Unit]("compile with dotty")
+lazy val dotcd = inputKey[Unit]("decompile tasty files")
+lazy val scalac = inputKey[Unit]("compile with scalac")
+lazy val runDotty = inputKey[Unit]("run a classfile with the dotty library available")
```

Comment:
These keys help for testing within the `tasty` project but maybe they are too polluting?

Reply:
These were removed in https://github.com/scala/scala/pull/8865/commits/d6cca665e19f146ded8d206feac6945fc9090756